sites/partners/cypress/fixtures/application.json

```diff
@@ -0,0 +1,76 @@
+{
+  "language": "English",
```

Comment:
I would prefer that this data be typed like we are doing in applicationData.ts in the public Cypress tests. We have a typed Application mock data in there so I think you can mostly copy/paste

Reply:
after talking with sean about this, since the paper application has no validation on it we do test variations where data that should be there is not there. When we add validation to these forms then I think we should definitely type our fields like that